include/wx/aui/auibar.h

```diff
@@ -345,7 +365,7 @@ class WXDLLIMPEXP_AUI wxAuiToolBarArt
 
     virtual int ShowDropDown(
                          wxWindow* wnd,
-                         const wxAuiToolBarItemArray& items) = 0;
+                         const wxAuiToolBarItemList& items) = 0;
```

Comment:
[It looks like](https://cs.github.com/?scopeName=All+repos&scope=&q=wxAuiToolBar+ShowDropDown+-path%3Awx%2Faui%2Fauibar.h+-path%3Awx%2Faui%2Fbarartmsw.h+-path%3Asrc%2Faui%2Fauibar.cpp+-path%3Asrc%2Faui%2Fbarartmsw.cpp) this change is going to break quite some code, as the existing overloads won't be called any longer.